documentation/docs/examples/point-to-point-measurement.mdx

```diff
@@ -0,0 +1,112 @@
+
+The tool provides ability to change measuring line width & color.
+
+<DemoWrapper name="Cognite3DViewerDemo" />
+
+## Point to point distance measurment
+To enter into point to point measurement mode, use `MeasurementTool.enterMeasurementMode()` which will have default line width & color.
+
+```jsx runnable
+// import { MeasurementTool } from '@cognite/reveal/tools';
+
+const measurementTool = new MeasurementTool(viewer);
+measurementTool.enterMeasurementMode();
+```
+### Exit measurement
+To exit from measurement mode, use `MeasurementTool.exitMeasurementMode()`.
+
+```jsx runnable
+// import { MeasurementTool } from '@cognite/reveal/tools';
+
+const measurementTool = new MeasurementTool(viewer);
+measurementTool.enterMeasurementMode();
+measurementTool.exitMeasurementMode();
+```
+
+### Set width and color of the measuring line
+To change width and color of the line, use `MeasurementTool.setLineOptions(options: MeasurementOptions, meshObject?: THREE.Mesh)`. MeasurementOptions expects
+  - `lineWidth` : number
+  - `color` : number
+
+```jsx runnable
+// import { MeasurementTool } from '@cognite/reveal/tools';
+
+const lineOptions = {
+  lineWidth: 3,
+  color: 0xff0000
+};
+const measurementTool = new MeasurementTool(viewer);
+measurementTool.enterMeasurementMode();
+measurementTool.setLineOptions(lineOptions);
+```
+
+It's also possible to apply custom label content for the measurment.
+The tool provides option of custom callback for the users. `MeasurementOptions.distanceToLabelCallback` can be
+used to trigger custom operation like changing the measuring units to feet:
+
+```jsx runnable
+// import { MeasurementTool } from '@cognite/reveal/tools';
+
+const measurementTool = new MeasurementTool(viewer, {distanceToLabelCallback: (distanceInMeters) => {
+      // 1 meter = 3.281 feet
+      return `${(distanceInMeters * 3.281).toFixed(2)} ft`;
+     }});
+measurementTool.enterMeasurementMode();
+```
+
+### Get all measurement
+To retrive all measurements added in the viewer, we can provide `MeasurementTool.getAllMeasurement()` which will
+returns array of meshes `THREE.Mesh[]`.
+
+```jsx
+// import { MeasurementTool } from '@cognite/reveal/tools';
+
+const measurementTool = new MeasurementTool(viewer);
+measurementTool.enterMeasurementMode();
+
+//get all measurements
+const measurements = measurementTool.getAllMeasurement();
+```
+
+### Remove measurement
+To remove a measurement from the viewer, we have `MeasurementTool.removeMeasurement(measurement: THREE.Mesh)`
+  - 'measurement` : Measurement mesh to be removed.
+
+```jsx
+// import { MeasurementTool } from '@cognite/reveal/tools';
+
+const measurementTool = new MeasurementTool(viewer);
+measurementTool.enterMeasurementMode();
+
+//Before retrival of measurements, add some measurement
+const measurements = measurementTool.getAllMeasurement();
+
+//Remove the first measurement added
+measurementTool.removeMeasurement(measurements[0]);
```

Comment:
I'd prefer if this was interactive - e.g. delete measurement after N seconds after its created.